packit/config/requirements.py

```diff
@@ -0,0 +1,22 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+from typing import Optional
+
+
+class LabelRequirementsConfig:
+    """Configuration of requirements on label."""
+
+    def __init__(
+        self,
+        present: Optional[list] = None,
+        absent: Optional[list] = None,
```

Comment:
I think you should be able to do:
```suggestion
        present: Optional[list[str]] = None,
        absent: Optional[list[str]] = None,
```